scripts/gulp/tasks/screener.ts

```diff
@@ -14,16 +15,26 @@ task('screener:runner', cb => {
   // screener-runner doesn't allow to pass custom options
   if (argv.filter) process.env.SCREENER_FILTER = argv.filter as string;
 
+  const changedPackages = getChangedPackages();
+  const screenerConfigPath = paths.base('scripts/screener/screener.config.js');
+
   // kill the server when done
-  screenerRunner(paths.base('scripts/screener/screener.config.js'))
-    .then(() => {
-      cb();
-      process.exit(0);
-    })
-    .catch(err => {
-      cb(err);
-      process.exit(1);
-    });
+  const handlePromiseExit = promise =>
+    promise
+      .then(() => {
+        cb();
+        process.exit(0);
+      })
+      .catch(err => {
+        cb(err);
+        process.exit(1);
+      });
+
+  if (changedPackages.has('@fluentui/docs')) {
```

Comment:
can we add some check that this package exists - if anyone would rename it Ci would continue to work "as expected"

Reply:
yeah that's a good point